nltools/tests/test_data.py

```diff
@@ -67,7 +67,7 @@ def test_brain_data_3mm(tmpdir):
     index = range(4)
     assert len(dat[index]) == len(index)
     index = dat.Y == 1
-    assert len(dat[index]) == index.values.sum()
+    assert len(dat[index.values.flatten()]) == index.values.sum()
```

Comment:
Let's keep the original test too as I want it to work with both pandas and numpy logical indexing.

add this back in `assert len(dat[index]) == index.values.sum()`

Reply:
Added back in. Also made sure all tests in 3mm are also in 2mm and visa versa. Looks like some were missing across the different versions.